client/src/users/connect.container.js

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux';
+import UserConnect from './connect.view';
+import * as duck from './duck';
+
+let timer = null;
+
+const mapDispatchToProps = dispatch => ({
+    onChangeValue: (evt) => {
+        clearTimeout(timer);
```

Comment:
what is this timer about?

Reply:
timeout not to be sued here, but if you need to do this check function `throttle`